config/introscope_agent.yml

```diff
@@ -15,6 +15,6 @@
 
 # Configuration for the CA Wily framework
 ---
-repository_root: ""
-version: 9.7.+
+repository_root: "https://bintray.com/ca/apm-agents/download_file?file_path="
```

Comment:
This should be the more direct URI `https://ca.bintray.com/apm-agents`.  The contents of the `index.yml` should look like the following, using those same more direct URIs:

```
---
9.7.0_27: https://ca.bintray.com/apm-agents/IntroscopeAgentFiles-NoInstaller9.7.0.27default.unix.tar
9.7.1_16: https://ca.bintray.com/apm-agents/IntroscopeAgentFiles-NoInstaller9.7.1.16default.unix.tar
10.0.0_16: https://ca.bintray.com/apm-agents/IntroscopeAgentFiles-NoInstaller10.0.0.16default.unix.tar
10.1.0_15: https://ca.bintray.com/apm-agents/IntroscopeAgentFiles-NoInstaller10.1.0.15default.unix.tar
10.2.0_27: https://ca.bintray.com/apm-agents/IntroscopeAgentFiles-NoInstaller10.2.0.27default.unix.tar
10.3.0_19: https://ca.bintray.com/apm-agents/IntroscopeAgentFiles-NoInstaller10.3.0.19default.unix.tar
10.5.0_20: https://ca.bintray.com/apm-agents/IntroscopeAgentFiles-NoInstaller10.5.0.20default.unix.tar
10.5.1_6: https://ca.bintray.com/apm-agents/IntroscopeAgentFiles-NoInstaller10.5.1.6default.unix.tar
10.5.2_15: https://ca.bintray.com/apm-agents/IntroscopeAgentFiles-NoInstaller10.5.2_15default.unix.tar
```